services/airlock/install.sh

```diff
@@ -3,7 +3,9 @@ set -euo pipefail
 
 DIR=~opensafely/airlock
 OVERRIDE_FILE="backends/$BACKEND/airlock.override.yaml"
```

Comment:
if we're running `load-env` should these come after that?

Reply:
I thought $BACKEND is a global env var from /etc/environment, and not in load-env?

As we set it with bootstrap.sh?